.github/labeler.yml

```diff
@@ -0,0 +1,141 @@
+# Add 'project' label to changes in basic project documentation and .github folder, excluding .github/workflows
+area/project:
+  - all:
+      - changed-files:
+          - any-glob-to-any-file:
+              - .github/**
+              - LICENSE
+              - AUTHORS
+              - MAINTAINERS
+              - PROJECT.md
+              - README.md
+              - .gitignore
+              - codecov.yml
+          - all-globs-to-all-files: "!.github/workflows/*"
+
+# Add 'ci' label to changes in the .github/workflows folder
+area/ci:
+  - changed-files:
+      - any-glob-to-any-file: ".github/workflows/**"
+
+# Add 'tests' label to changes in test files
+area/testing:
+  - changed-files:
+      - any-glob-to-any-file:
+          - tests/**
+          - "**/*_test.go"
+
+# area:api
+area/api:
+  - changed-files:
+      - any-glob-to-any-file: "api/**"
+
+# cache
+area/cache:
+  - changed-files:
+      - any-glob-to-any-file: "cache/**"
+
+# area:cache/remote
+area/cache/remote:
+  - changed-files:
+      - any-glob-to-any-file: "cache/remotecache/**"
+
+# area:buildctl
+area/buildctl:
+  - changed-files:
+      - any-glob-to-any-file: "cmd/buildctl/**"
+
+# area:buildkitd
+area/buildkitd:
+  - changed-files:
+      - any-glob-to-any-file: "cmd/buildkitd/**"
+
+# area:dependencies
+area/dependencies:
+  - changed-files:
+      - any-glob-to-any-file:
+          - go.mod
+          - go.sum
+          - vendor/**
+          - tools/**
+
+# area:docs
+area/docs:
+  - changed-files:
+      - any-glob-to-any-file:
+          - "docs/**"
+          - "frontend/dockerfile/docs/**"
+
+# area:dockerfile
+area/dockerfile:
+  - changed-files:
+      - any-glob-to-any-file: "frontend/dockerfile/**"
+
+# area:dockerfile/checks
+area/dockerfile/checks:
+  - changed-files:
+      - any-glob-to-any-file:
+          - "frontend/dockerfile/linter"
+          - "frontend/dockerfile/subrequests/lint"
+
+# area:examples
+area/examples:
+  - changed-files:
+      - any-glob-to-any-file: "examples/**"
+
+# area:executor
+area/executor:
+  - changed-files:
+      - any-glob-to-any-file: "executor/**"
+
+# area:exporter
+area/exporter:
+  - changed-files:
+      - any-glob-to-any-file: "exporter/**"
+
+# area:frontend
+area/frontend:
+  - changed-files:
+      - any-glob-to-any-file: "frontend/**"
+
+# area:hack
+area/hack:
+  - changed-files:
+      - any-glob-to-any-file: "hack/**"
+
+# area:session
+area/session:
+  - changed-files:
+      - any-glob-to-any-file: "session/**"
+
+# area:snapshot
+area/snapshot:
```

Comment:
this can be connected with `cache` . Or we can create `area/storage` that has cache/* (except remotecache) and `snapshot/*` . Possibly there is a case for reorganizing packages here as well.

Reply:
moved `cache` and `snapshot` under `area/storage` with an exclusion for `cache/remotecache` which has it's own label `area/remotecache`